packages/content-management/content_insights/content_insights_public/src/components/views_stats/views_chart.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { Chart, Settings, DARK_THEME, LIGHT_THEME, BarSeries, Axis } from '@elastic/charts';
+import { formatDate, useEuiTheme } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import moment from 'moment';
+
+const dateFormatter = (d: Date) => formatDate(d, `MM/DD`);
+
+const seriesName = i18n.translate('contentManagement.contentEditor.viewsStats.viewsLabel', {
+  defaultMessage: 'Views',
+});
+
+const weekOfFormatter = (date: Date) =>
+  i18n.translate('contentManagement.contentEditor.viewsStats.weekOfLabel', {
+    defaultMessage: 'Week of {date}',
+    values: { date: dateFormatter(date) },
+  });
+
+export const ViewsChart = ({ data }: { data: Array<[week: string, views: number]> }) => {
+  const { colorMode } = useEuiTheme();
+
+  const momentDow = moment().localeData().firstDayOfWeek();
+  const isoDow = momentDow === 0 ? 7 : momentDow;
+
+  return (
+    <Chart size={{ height: 240 }}>
+      <Settings
+        baseTheme={colorMode === 'DARK' ? DARK_THEME : LIGHT_THEME}
+        showLegend={false}
+        dow={isoDow}
+      />
+      <BarSeries
+        id="viewsOverTime"
+        name={seriesName}
+        data={data.map(([week, views]) => [new Date(week).getTime(), views])}
+        xAccessor={0}
+        yAccessors={[1]}
+        enableHistogramMode={true}
+        yNice={true}
+        minBarHeight={1}
+        xScaleType="time"
+        timeZone="utc"
```

Comment:
Thanks for the review! I thought that by sticking to UTC I could keep it more streamlined for me. 
So I noticed that moment is already configured with the timezone from advanced settings and I removed all .UTC() conversion from client code and pass moment.tz to the chart. Does it seem that it uses the correct timezone now?

Reply:
Yeah, that's much better. I haven't seen `moment().tz()` used anywhere to pull the timezone but it seems to work the same so I don't think it's an issue.

Since this is by week the timezone is not as noticeable as it would be with hours, but to be safe using the default timezone is best. If someone were to look at the dashboard on the end of a week and the `utc` was applied, it could show up as a stat in the current week.

Reply:
thanks, it is very cumbersome to pull in uiSettings dependency directly :(